docs/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,30 @@
+:pyenv: link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.4.0b/src/openzeppelin/access/ownable/library.cairo[pyenv]
+
+= Overview
+
+Nile is a CLI tool to develop or interact with StarkNet projects written in Cairo. It consists of different components for developing, compiling, testing, and deploying your smart contracts and dApps, providing a CLI for executing tasks, and a Runtime Environment (NRE) for scripting.
+
+The package is designed to be extensible and very customizable by using plugins. This guide will take you through the installation of our recommended setup, but as we expect a lot of functionality coming from plugins, you are free to customize it.
+
+TIP: You can get a list of plugins extending Nile functionality from the xref:plugins.adoc[Plugins] section.
+
+== Requirements
+
+=== GMP for fastecdsa
```

Comment:
i believe this is an installation step so it probably belongs to quickstart. it makes me doubt this overview section should be a different one at all. what do you think about merging them?

tbh the gmp/fastecdsa issue is not a very sexy first impression for the docs

Reply:
Agree. I refactored it into just Overview.